index.html

```diff
@@ -100,6 +106,221 @@ <h1>I'm Nasir Glad to see you!</h1>
+              <h2>Multi Post Stories</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <div class="portfolio-card">
+            <div class="portfolio-thumbnail">
+              <img src="images/work3.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Tonic</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <div class="portfolio-card">
+            <div class="portfolio-thumbnail">
+              <img src="images/work4.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Multi Post Stories</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </section>
+      <!-- End portfolio section -->
+      <!-- Start About section -->
+      <section id="about">
+        <div class="container">
+          <div class="about-content">
+            <div class="about-main-content">
+              <h2>About Myself</h2>
+              <p>
+                I'm a software developer! I can help you build a project,
+                feature or website look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
+            </div>
+            <!-- End hero main content section -->
+            <!-- Start hero social section -->
```

Comment:
- [ ] I suggest adding the border top right radius right as per the Figma design as we have to stick to it as much as possible :recycle: 

Your design:
![Screenshot from 2023-03-24 19-00-43](https://user-images.githubusercontent.com/97834160/227592044-8ff24d31-7abe-4038-aea3-7cfd687f9645.png)

Figma:

![Screenshot from 2023-03-24 18-58-37](https://user-images.githubusercontent.com/97834160/227591719-198c1ddd-4f14-4aca-ab70-8bdc8e4e2ffc.png)